<?xml version="1.0"?>
<!-- Launches the MPC controller ROS node. -->

<launch>
  <arg name="config"/>
  <arg name="ctrl_config" default="default" doc="alternative controller config overwriting the one in config"/>
  <arg name="planner_config" default="default" doc="alternative planner config overwriting the one in config"/>
  <arg name="logging_sub_folder" default="default" doc="save data to a sub folder if set"/>

  <!-- Controller -->
  <node pkg="mm_run" type="mpc_ros.py" name="controller_mpc" output="screen" required="true" args="--config $(arg config) --ctrl_config $(arg ctrl_config) --planner_config $(arg planner_config) --logging_sub_folder $(arg logging_sub_folder)"/>

</launch>
